packages/contracts-bedrock/src/L1/L1ERC721Bridge.sol

```diff
@@ -110,7 +123,7 @@ contract L1ERC721Bridge is ERC721Bridge, ISemver {
         IERC721(_localToken).transferFrom(_from, address(this), _tokenId);
 
         // Send calldata into L2
-        MESSENGER.sendMessage(OTHER_BRIDGE, message, _minGasLimit);
+        messenger.sendMessage(otherBridge, message, _minGasLimit);
```

Comment:
Can you also update this to using kwargs style syntax for both `sendMessage` and `transferFrom`? Refactoring as we touch code to improve code quality, our style guide has since updated to prefer this style especially when there is more than 2 args